shared/js/background/chrome-events.es6.js

```diff
@@ -850,10 +858,7 @@ chrome.alarms.onAlarm.addListener(alarmEvent => {
     } else if (alarmEvent.name === 'clearExpiredHTTPSServiceCache') {
         httpsService.clearExpiredCache()
     } else if (alarmEvent.name === 'updateUserAgentData') {
```

Comment:
I don't know when/if user agent rotation is coming back, can we drop this if and corresponding chrome.alarm ?